### DISCUSSION TITLE
Services common to number of providers ; Providers attached to different plans

### DISCUSSION THREAD

**Discussion by sharma-TIP** [CMSgov/price-transparency-guide#358] [Ideas]
A group of services same rates can be common to 100's of providers ; These providers can be attached to 100's of plan. If we use multiple plan approach then because of the provider array being in the Multiple plan in-network file JSON - We would have to create 100's of these files for each provider network - Creating a lot of redundancy / repeatable data. 

Example 
Services - S1,S2,S3,S4 at 60% of billed charges are common to Provider P1,P2,P3,P4...P100. 

Each provider can be in multiple plans; to use Multiple plans per file schema we would have to generate 100 such files to refer them uniquely in the TOC files.

If the Provider reference can be moved to TOC level then it would be easier for everyone to group common services at same rate and reference the file in TOC. This means that multiple plans per file the provider reference will become optional.  

Example: 
Current Schema
{
       "plan_name": "medicare",
       "plan_id_type": "hios",
       "plan_id": "0000000000",
       "plan_market_type": "individual"
     }],
     "in_network_files": [{
       "description": "in-network file",
       "location": "https://www.some_site.com/files/in-network-file-123456.json"
     },{
       "description": "behavioral in-network shared file",
       "location": "https://www.some_site.com/files/behavioral-health-0000.json"
     }],

Add Provider Reference object to TOC Json; now I can refer the in network file across multiple files and just change the Provider reference ; This will solve a lot of data redundancy. 
{
       "plan_name": "medicare",
       "plan_id_type": "hios",
       "plan_id": "0000000000",
       "plan_market_type": "individual"
     }],
     "in_network_files": [{
       "description": "in-network file",
       "provider_reference" :  "https://www.some_site.com/files/pvd_ref-123456.json"
       "location": "https://www.some_site.com/files/in-network-file-123456.json"
     },{
       "description": "behavioral in-network shared file",
       "location": "https://www.some_site.com/files/behavioral-health-0000.json"
     }],
@shaselton-usds

**Comment by shaselton-usds** [comment]
@sharma-TIP I think I understand the intent. So, would the `https://www.some_site.com/files/in-network-file-123456.json` have no provider network information in it because of the assumed `https://www.some_site.com/files/pvd_ref-123456.json`?

**Reply by sharma-TIP** [reply]
Yes, Sir. Then we can attach that service, rate JSON files to N number of Plans ( wherever applicable) without changing the JSON file.
